app/pages/project/instances/InstancesPage.tsx

```diff
@@ -70,6 +66,13 @@ export function InstancesPage() {
     query: { project, limit: PAGE_SIZE },
   })
 
+  // if any instance in the list is transitioning, poll
+  // TODO: figure out this logic. polling if any instance is transitioning might be excessive
+  useInterval({
+    fn: () => apiQueryClient.invalidateQueries('instanceList'),
+    delay: instances.items.some(instanceTransitioning) ? 1000 : null,
+  })
+
```

Comment:
Not sure if polling when any instance is transitioning is a good idea. Might make more sense to just always poll but on a slower interval like 10s.